Dashboard/app/js/lib/components/devices/AssignmentsEditView/screens/AssignDatapoints/index.jsx

```diff
@@ -119,10 +124,10 @@ export default class AssignDatapoints extends React.Component {
             {allDPAssigned ? (
               <AllDatapoints deviceId={deviceData.id} />
             ) : (
-              <DatapointList datapointsData={datapointsData} />
+              <DatapointList datapointsData={dataPointsWithDetails} />
             )}
 
-            {!allDPAssigned && hasMore && (
+            {!!showLoadMore && (
```

Comment:
`showLoadMore` is already a boolean right? Is the `!!` needed?

Reply:
It's not a boolean it's a falsy number (in this case 0)